doc/replication.md

```diff
@@ -98,8 +98,12 @@ CDTReplicator *replicator = [replicatorFactory oneWay:pushReplication error:&err
 
 // Check replicator isn't nil, if so check error
 
-// Start the replication and wait for it to complete
-[replicator start];
+// Start the replication
+[replicator startWithError:&error];
+
+//check replication started
```

Comment:
Comments don't match the code, since the document is supposed to illustrate best practice, you should check and exit with a log message if there was a problem.